tests/test_schedule.c

```diff
@@ -170,27 +177,27 @@ void test_simple_case( void )
 
     print_schedule( s );
 
-    block = get_blocked_at_time( s, 1233999 );
+    block = get_blocked_at_time( s, 1233999, &next_unixtime );
```

Comment:
You should verify the `next_unixtime` value is correct.

Reply:
Done.